core/time/now_spec.rb

```diff
@@ -3,4 +3,49 @@
 
 describe "Time.now" do
   it_behaves_like :time_now, :now
+
+  describe ":in keyword argument" do
```

Comment:
Note: I first thought those spec could be put into `core/time/shared/now.rb` as shared examples. But the `in` keyword behavior differs between `Time.now` and `Time.new` by Ruby version, so I just wrote these specs without `ruby_version_is` here.